package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cssfmt",
-  "version": "2.1.4",
+  "version": "2.1.5",
```

Comment:
this changes will not release as patch, i wanna to release it as the next major release, and i will rename cssfmt to stylefmt, so you don't have to update `version` in `package.json` and `CHANGELOG.md`.